docs/internals.rst

```diff
@@ -0,0 +1,332 @@
+.. _internals:
+
+Internals
+=========
+
+Dredd itself is a `command-line <https://en.wikipedia.org/wiki/Command-line_interface>`__ `Node.js <https://nodejs.org/>`__ application written in modern JavaScript.
+
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+Maintainers
+-----------
+
+`Apiary <https://apiary.io/>`__ is the main author and maintainer of Dredd’s `upstream repository <https://github.com/apiaryio/dredd>`__. Currently responsible people are:
+
+
+-  `@paraskakis <https://github.com/paraskakis>`__ - product decisions, feature requests
+-  `@honzajavorek <https://github.com/honzajavorek>`__ - lead of development
+
+Dredd supports so many programming languages only thanks to the work of several contributors, who were so awesome they dedicated time to create, improve, and maintain the respective :ref:`hook handlers <hooks>`:
```

Comment:
This paragraph is a little awkward and I am not sure how to completely fix it.

I'd suggest changing the first part to:

> Dredd supports many programming languages thanks to the work of several contributors

The part "who were so awesome" feels off to me but I don't have a suggestion on how to improve it.